### ISSUE TITLE
VsCode not responsive since last update when typing TypeScript code

### ISSUE THREAD

**Issue by MrDesjardins** [Microsoft/vscode#28646]
- VSCode Version: Code 1.13.0 (376c52b955428d205459bea6619fc161fc8faacf, 2017-06-08T16:43:13.058Z)
- OS Version: Windows_NT ia32 10.0.15063
- Extensions: 

|Extension|Author|Version|
|---|---|---|
|Bookmarks|alefragnani|0.15.1|
|bracket-pair-colorizer|CoenraadS|0.10.7|
|tslint|eg2|0.15.0|
|vscode-codemetrics|kisstkondoros|1.7.2|
|debugger-for-chrome|msjsdiag|3.1.3|
|vscode-subword-navigation|ow|1.1.5|
|vscode-icons|robertohuertasm|7.9.0|
|gitblame|waderyan|1.11.2|;

---

Steps to Reproduce:

1. Typing code (TS)
2. Application hand up. This started to occurs since the last update. I can resize the windows, go in the main top menu, but cannot click in the code, or click anything on the left side. I can click the 'X' but nothing happen.

I don't have more accurate steps to reproduce. You can reach my at padesjar@microsoft.com if we you want to chat about it.

**Comment by mjbvz** [comment]
I'm unable to reproduce this. First can you please make sure an extension is not to blame by starting code with the `--disable-extensions` command line flag

**Comment by steelsojka** [comment]
This may be related to this issue https://github.com/Microsoft/vscode/issues/28824

**Comment by MrDesjardins** [comment]
For your information, I didn't see the error today, neither the day before. However, since then, I worked on smaller file (always under 1000 lines of code). When it was crashing, I also noticed that it was happening not only when working in some TypeScript file, but happened also when executing some searches on a big repository without a filter for file extension. Maybe it's not related, I am still trying to find the pattern.
